For me compilation fails here:

```
[ 55%] Performing build step for 'openblas-git'
make[4]: warning: -jN forced in submake: disabling jobserver mode.
...
make[4]: warning: -jN forced in submake: disabling jobserver mode.
Undefined symbols for architecture x86_64:
  "_LAPACKE_cbbcsd", referenced from:
     -exported_symbol[s_list] command line option
... many more of those
  "_sgemm_incopy", referenced from:
      _sgemm_tn in libopenblas_sandybridgep-r0.2.8.a(sgemm_tn.o)
      _sgemm_tt in libopenblas_sandybridgep-r0.2.8.a(sgemm_tt.o)
      _strmm_LTUU in libopenblas_sandybridgep-r0.2.8.a(strmm_LTUU.o)
      _strmm_LTUN in libopenblas_sandybridgep-r0.2.8.a(strmm_LTUN.o)
      _strmm_LTLU in libopenblas_sandybridgep-r0.2.8.a(strmm_LTLU.o)
      _strmm_LTLN in libopenblas_sandybridgep-r0.2.8.a(strmm_LTLN.o)
      _strsm_LTUU in libopenblas_sandybridgep-r0.2.8.a(strsm_LTUU.o)
... some more of that, too
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[4]: *** [libopenblas_sandybridgep-r0.2.8.dylib] Error 1
make[3]: *** [shared] Error 2

```

Your commit befa06fc9bae896e553b501e0f775d18055ee032 also changed the git version to head and added the buildem env, I tried several combinations to be sure, but it is always TARGET=SANDYBRIDGE that makes it fail. Without that, it builds fine here!

It seems that our processors are so different that OpenBLAS can correctly identify mine, but doesn't recognize yours. Did you try to set `TARGET=HASWELL`? That one works fine for me (found it here https://github.com/xianyi/OpenBLAS/issues/332).

What does the following command output for you?

```
$ sysctl -n machdep.cpu.brand_string
  Intel(R) Core(TM) i7-4558U CPU @ 2.80GHz
```

--- a/README_openblas.txt
+++ b/README_openblas.txt
@@ -1,0 +1,7 @@
+If you are having trouble building OpenBLAS out of the box, you may need to specify your CPU Architecture.
+Common architecture options include SandyBridge and Haswell. They can be specified by using the TARGET flag.
+For example, TARGET=SANDYBRIDGE will build for the SandyBridge architecture. Similarly, TARGET=HASWELL will
+target the Haswell (or Crystalwell) architecture(s). More information can be found on the OpenBLAS GitHub
+webpage located ( https://github.com/xianyi/OpenBLAS/ ) here. To do this simply modify the BUILD_COMMAND to
+include this flag ____after____ the call to $(MAKE) as it must be an argument to make. It is recommend the
+same be done for the INSTALL_COMMAND as well.

--- a/README_openblas.txt
+++ b/README_openblas.txt
@@ -1,7 +1,15 @@
 If you are having trouble building OpenBLAS out of the box, you may need to specify your CPU Architecture.
-Common architecture options include SandyBridge and Haswell. They can be specified by using the TARGET flag.
-For example, TARGET=SANDYBRIDGE will build for the SandyBridge architecture. Similarly, TARGET=HASWELL will
-target the Haswell (or Crystalwell) architecture(s). More information can be found on the OpenBLAS GitHub
-webpage located ( https://github.com/xianyi/OpenBLAS/ ) here. To do this simply modify the BUILD_COMMAND to
-include this flag ____after____ the call to $(MAKE) as it must be an argument to make. It is recommend the
-same be done for the INSTALL_COMMAND as well.
+Common architecture options include SandyBridge and Haswell. Even if your build worked correctly, it may
+not be optimal. The default architecture, if not specified, is Nehalem, which is probably not the ideal
+choice unless using an older computer.
+
+To specify the architecture simply use the TARGET flag. For example, TARGET=SANDYBRIDGE will build for
+the SandyBridge architecture. Similarly, TARGET=HASWELL will target the Haswell (or Crystalwell)
+architecture(s). To do this simply modify the BUILD_COMMAND to include this flag ____after____ the call
+to $(MAKE) as it must be an argument to make. It is recommend the same be done for the INSTALL_COMMAND
+as well.
+
+More information can be found on the OpenBLAS GitHub webpage ( https://github.com/xianyi/OpenBLAS/ ). A list
+of architectures supported can be found here ( https://github.com/xianyi/OpenBLAS/blob/develop/TargetList.txt ).
+Note that at the time of writing this READNME, the list has not been updated in about a year; so, it does not
+list HASWELL even though it is supported by the version we are using.

--- a/openblas.cmake
+++ b/openblas.cmake
@@ -10,8 +10,7 @@ include (ExternalProject)
 include (ExternalSource)
 
 external_git_repo (openblas
-    #v0.2.8
-    HEAD
+    4d42368214f2fd102b2d163c086e0f2d8c166dc6
     https://github.com/xianyi/OpenBLAS)
 
 message ("Installing ${openblas_NAME} into ilastik build area: ${BUILDEM_DIR} ...")
@@ -23,11 +22,11 @@ ExternalProject_Add(${openblas_NAME}
     UPDATE_COMMAND      ""
     PATCH_COMMAND       ""
     CONFIGURE_COMMAND   ""
-    # Added Sandybridge target. This is required for SandyBridge architecture. It also will optimize for Haswell architechture, which includes Clearwell.
-    BUILD_COMMAND       ${BUILDEM_ENV_STRING} $(MAKE) NO_AVX=1 NO_AFFINITY=1 TARGET=SANDYBRIDGE -j8
+    # Add TARGET=xxx (where xxx is an architecture) to the BUILD_COMMAND to optimize and/or avoid particular build errors. See README_openblas.txt for details.
+    BUILD_COMMAND       ${BUILDEM_ENV_STRING} $(MAKE) NO_AVX=1 NO_AFFINITY=1 -j8
     BUILD_IN_SOURCE     1
-    # Added Sandybridge target. This is required for SandyBridge architecture. It also will optimize for Haswell architechture, which includes Clearwell.
-    INSTALL_COMMAND     ${BUILDEM_ENV_STRING} $(MAKE) PREFIX=${BUILDEM_DIR} TARGET=SANDYBRIDGE install &&
+    # Add TARGET=xxx (where xxx is an architecture) to the BUILD_COMMAND to optimize and/or avoid particular build errors. See README_openblas.txt for details.
+    INSTALL_COMMAND     ${BUILDEM_ENV_STRING} $(MAKE) PREFIX=${BUILDEM_DIR} install &&
                         ln -fs libopenblas.so ${BUILDEM_DIR}/lib/libblas.so &&
                         ln -fs libopenblas.so ${BUILDEM_DIR}/lib/liblapack.so
 )

--- a/vtk.cmake
+++ b/vtk.cmake
@@ -81,6 +81,10 @@ ExternalProject_Add(${vtk_NAME}
     -DZLIB_INCLUDE_DIR=${BUILDEM_INCLUDE_DIR}
     -DZLIB_LIBRARY=${BUILDEM_LIB_DIR}/libz.${BUILDEM_PLATFORM_DYLIB_EXTENSION}
     
+    # XCode Tools 5.1 moving forward no longer supports garbage collection ( -fobjc-gc flag ). However, Mac, AFAIK
+    # still supports Manual and Automatic Reference Counting. By setting VTK_REQUIRED_OBJCXX_FLAGS to "", manual
+    # reference counting is used by VTK, which should work on other Mac OS versions.
+    -DVTK_REQUIRED_OBJCXX_FLAGS=""
     
 	# We want vtk to be built in parallel if possible.
 	# Therefore we use $(MAKE) instead of 'make', which somehow enables sub-make files to use the jobserver correctly.

